src/core/fusebox.ts

```diff
@@ -41,28 +30,32 @@ export function fusebox(config: IPublicConfig) {
     }
   }
   return {
-    runDev: async (props: IRunDevProps) => {
-      const ctx = createContext(config);
-      ctx.creatOutputConfig(props.bundles);
+    runDev: async (props?: IRunProps) => {
+      const ctx = new Context(config, props);
```

Comment:
I'd really avoid calling new Context, let's keep `createContext` and it will be easier to refactor. Let's just have one `createContext`